README.md

```diff
@@ -1,11 +1,33 @@
-# lambda-template
-A GitHub template for quickly starting a new AWS lambda project.
-
-## Naming
-Naming conventions:
-* for a vanilla Lambda: `lambda-<context>`
-* for a Cloudformation Transform macro: `cfn-macro-<context>`
-* for a Cloudformation Custom Resource: `cfn-cr-<context>`
+# lambda-finops-cost-rules
+A lambda for generating the JSON string required by AWS::CE::CostCategory.
+
+## Design
+This lambda is intended to transform the chart of accounts returned from
+[lambda-mips-api](https://github.com/Sage-Bionetworks-IT/lambda-mips-api),
+combined with current AWS account tags, into cost-category rules (JSON string)
+appropriate to pass to the Rules parameter of AWS::CE::CostCategory.
```

Comment:
this is a bit too abstract for me to grok.  would you be able to provide some examples of `the chart of accounts returned from lambda-mips-api` and examples of the cost-category rules that `https://abcxyz.cloudfront.net/rules` would return?